ext/gd/tests/webp_basic.phpt

```diff
@@ -29,11 +31,19 @@ imagewebp($im1, $filename);
 
 $im2 = imagecreatefromwebp($filename);
 imagewebp($im2, $filename);
+echo 'Is lossy conversion close enough? ';
 var_dump(calc_image_dissimilarity($im1, $im2) < 10e5);
+
+imagewebp($im1, $filename, WEBP_LOSSLESS);
+$im_lossless = imagecreatefromwebp($filename);
+echo 'Does lossless conversion work? ';
+var_dump(calc_image_dissimilarity($im1, $im_lossless) == 0);
+
 ?>
 --CLEAN--
 <?php
 @unlink(__DIR__ . '/webp_basic.webp');
 ?>
 --EXPECT--
-bool(true)
+Is lossy conversion close enough? bool(true)
+Does lossless conversion work? bool(true)
```

Comment:
Maybe one more test case for false makes sense

Reply:
> Maybe one more test case for false makes sense

What sort of test case are you thinking of?

Actually, I wonder sort of the opposite... is the lossy conversion test still necessary?